UPGRADING.md

```diff
@@ -1,10 +1,64 @@
 Upgrading Grape
 ===============
 
+### Upgrading to >= 1.5.4
```

Comment:
Unwrap lines, we github markdown here.

Reply:
Will do if you like - I'm a 80 chars person :D